src/main/java/enums/GameSelections.java

```diff
@@ -0,0 +1,48 @@
+package enums;
+
+public enum GameSelections {
+    SCISSOR(0, "가위"),
+    ROCK(1, "바위"),
+    PAPER(2, "보");
+
+    private final int selectNumber;
+    private final String selections;
+
+    GameSelections(int selectNumber, String selections) {
+        this.selectNumber = selectNumber;
+        this.selections = selections;
+    }
+
+    public String getValue() {
+        return selections;
+    }
+
+    public static GameSelections getGameSelection(int selectNumber) {
+        if (selectNumber == SCISSOR.selectNumber) {
+            return SCISSOR;
+        }
+        if (selectNumber == ROCK.selectNumber) {
+            return ROCK;
+        }
+        if(selectNumber == PAPER.selectNumber) {
+            return PAPER;
+        }
+        throw new RuntimeException("올바른 선택지가 아닙니다.");
```

Comment:
이 부분은 아래와 같이 변경해볼 수 있을 것 같아요.
```suggestion
        for (GameSelections gameSelections : GameSelections.values()) {
            if (gameSelections.selectNumber == selectNumber) {
                return gameSelections;
            }
        }
        throw new RuntimeException("올바른 선택지가 아닙니다.");
```